ts/screens/wallet/TransactionDetailsScreen.tsx

```diff
@@ -314,7 +350,8 @@ class TransactionDetailsScreen extends React.Component<Props> {
 }
 
 const mapDispatchToProps = (dispatch: Dispatch) => ({
-  navigateToWalletHome: () => dispatch(navigateToWalletHome()),
+  navigateBackToEntrypointPayment: () => dispatch(backToEntrypointPayment()),
+  popHistory: () => dispatch(navigationHistoryPop(3)),
```

Comment:
avoid [magic number ](https://www.quora.com/What-is-magic-number)
It could be a const on top of this file

Reply:
It's a misprint the popHistory is no more used, thanks to `navigateBackToEntrypointPayment` action. 

I'll remove that in the next commit, thanks for noticing!